schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -40,6 +40,8 @@
                             },
                             {
                                 "oneOf": [
+                                    { "$ref": "http://schema.management.azure.com/schemas/2014-05-21-preview/Microsoft.DevTestLabs.json#/definitions/labs" },
```

Comment:
Please also update 2015-01-01\deploymentTemplate.json file to include the references.